paddle/operators/math/math_function.cu

```diff
@@ -273,6 +273,35 @@ void set_constant_with_place<platform::CUDAPlace>(
                            TensorSetConstantGPU(context, tensor, value));
 }
 
+template <typename T>
+__global__ void RowwiseAddKernel(const T* a, const T* b, T* c, int64_t width,
+                                 int64_t num) {
+  T tmp = 1.0 / width;
+  for (int i = blockIdx.x * blockDim.x + threadIdx.x; i < num;
```

Comment:
It would be better to change the type of `num` to int. Otherwise, there is a comparison of `int` data and `int64_t` data.

Reply:
Done. Thanks!